cadc-uws-server/src/test/java/ca/nrc/cadc/uws/server/impl/InitDatabaseUWSTest.java

```diff
@@ -0,0 +1,158 @@
+*  OpenCADC project.                    OpenCADC.
+*
+*  OpenCADC is free software:           OpenCADC est un logiciel libre ;
+*  you can redistribute it and/or       vous pouvez le redistribuer ou le
+*  modify it under the terms of         modifier suivant les termes de
+*  the GNU Affero General Public        la “GNU Affero General Public
+*  License as published by the          License” telle que publiée
+*  Free Software Foundation,            par la Free Software Foundation
+*  either version 3 of the              : soit la version 3 de cette
+*  License, or (at your option)         licence, soit (à votre gré)
+*  any later version.                   toute version ultérieure.
+*
+*  OpenCADC is distributed in the       OpenCADC est distribué
+*  hope that it will be useful,         dans l’espoir qu’il vous
+*  but WITHOUT ANY WARRANTY;            sera utile, mais SANS AUCUNE
+*  without even the implied             GARANTIE : sans même la garantie
+*  warranty of MERCHANTABILITY          implicite de COMMERCIALISABILITÉ
+*  or FITNESS FOR A PARTICULAR          ni d’ADÉQUATION À UN OBJECTIF
+*  PURPOSE.  See the GNU Affero         PARTICULIER. Consultez la Licence
+*  General Public License for           Générale Publique GNU Affero
+*  more details.                        pour plus de détails.
+*
+*  You should have received             Vous devriez avoir reçu une
+*  a copy of the GNU Affero             copie de la Licence Générale
+*  General Public License along         Publique GNU Affero avec
+*  with OpenCADC.  If not, see          OpenCADC ; si ce n’est
+*  <http://www.gnu.org/licenses/>.      pas le cas, consultez :
+*                                       <http://www.gnu.org/licenses/>.
+*
+************************************************************************
+*/
+
+package ca.nrc.cadc.uws.server.impl;
+
+import ca.nrc.cadc.db.version.InitDatabase;
+import ca.nrc.cadc.util.Log4jInit;
+import java.util.List;
+import org.apache.log4j.Level;
+import org.apache.log4j.Logger;
+import org.junit.Assert;
+import org.junit.Test;
+
+/**
+ *
+ * @author pdowler
+ */
+public class InitDatabaseUWSTest {
+    private static final Logger log = Logger.getLogger(InitDatabaseUWSTest.class);
+    
+    static {
+        Log4jInit.setLevel("ca.nrc.cadc.uws.server.impl", Level.INFO);
+        Log4jInit.setLevel("ca.nrc.cadc.db.version", Level.INFO);
+    }
+    
+    public InitDatabaseUWSTest() { 
+    }
+    
+    @Test
+    public void testParseCreateDDL() {
+        try {
+            InitDatabaseUWS init = new InitDatabaseUWS(null, null, "uws");
+            for (String fname : InitDatabaseUWS.CREATE_SQL) {
+                log.info("process file: " + fname);
+                List<String> statements = init.parseDDL(fname, "uws");
+                Assert.assertNotNull(statements);
+                Assert.assertFalse(statements.isEmpty());
+                /*
+                // sanity check statements
```

Comment:
Dead code or uncomment?

Reply:
This was code I had in caom2persistence, where I first implemented automating this and was paranoid about accidentally adding dangerous SQL commands... not sure if I'll enable or remove it here.

Reply:
This was code I had in caom2persistence, where I first implemented automating this and was paranoid about accidentally adding dangerous SQL commands... not sure if I'll enable or remove it here.